### PR TITLE
Add sspecacct to account passthrough

### PR DESCRIPTION
**Pull request by jmongerlyra**
Add `sspecacct` to account passthrough list.

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ vars:
     netsuite2_transactions: "{{ ref('stg_netsuite2__transactions') }}"
     netsuite2_vendor_categories: "{{ ref('stg_netsuite2__vendor_categories') }}"
     netsuite2_vendors: "{{ ref('stg_netsuite2__vendors') }}"
-    accounts_pass_through_columns: []
+    accounts_pass_through_columns: [sspecacct]
```
Comment (rwang-lyra) [inline dbt_project.yml:60]: I have renamed it `sspecacct as special_account_type_id,` at upstream, per the naming conventions

Comment (jmongerlyra) [reply dbt_project.yml:60]: Sorry, I'm not following. I do not see this field in `STG_NETSUITE2__ACCOUNTS` after the tables are refreshed. I ran `dbt clean`, `dbt deps`, then `dbt run -f`.

Comment (rwang-lyra) [reply dbt_project.yml:60]: https://github.com/rwang-lyra/dbt_netsuite_source_fork/blob/main/models/netsuite2/stg_netsuite2__accounts.sql#L30
here

Comment (jmongerlyra) [reply dbt_project.yml:60]: Got it. I see the renamed field now in `STG_NETSUITE2__ACCOUNTS`.

**Comment by jmongerlyra** [comment]
Duplicated by https://github.com/rwang-lyra/dbt_netsuite_fork/pull/8

**Pull request closed without merging**